### PR TITLE
Make Teacher Homepage components more narrow 

### PR DESCRIPTION
**Pull request by Erin007**
To better align the components and to better fit in the studio.code.org layout, I resized components that will be used for the Teacher Homepage. This PR also shows the Sections Table in context of the rest of the page rather than the Set Up Message. 

teacher-homepage-layout

### CODE REVIEW

**Comment by Erin007** [comment]
cc: @joshlory

File: apps/src/templates/teacherHomepage/CollapsibleSection.jsx [hunk 1]
```diff
@@ -72,13 +72,24 @@ const CollapsibleSection = React.createClass({
...
-            return (
-              <div key={index}>
-                {child}
-              </div>
-            );
+            if (index % 2 === 0) {
+              return (
+                <div key={index}>
+                  {child}
+                    <div style={styles.spacer}>
```
Comment (Erin007) [inline apps/src/templates/teacherHomepage/CollapsibleSection.jsx]: This works the way I want it to, but seems a little funky. Is there a better way to handle this?

File: apps/src/templates/teacherHomepage/CollapsibleSection.jsx [hunk 2]
```diff
@@ -72,13 +72,24 @@ const CollapsibleSection = React.createClass({
...
+                    </div>
+                </div>
+              );
+            } else {
+              return (
+                <div key={index}>
+                  {child}
+                </div>
+              );
+            }
```
Comment (Bjvanminnen) [inline apps/src/templates/teacherHomepage/CollapsibleSection.jsx]: Could simplify this by having just `{(index % 2 === 0) && <div style={styles.spacer}>.</div>}`

**Comment by tanyaparker** [comment]
1. The teacher announcement button isn't lining up with the text.
image


2. It doesn't look like it's using Gotham? When I look at the page itself and deselect the font-family attribute, it counterintuitively looks more Gotham.

What you currently have (not Gotham):
image

This is Gotham:
image

3. The course and resource cards are not quite the same width.

image

image

4. Can you update the little blue links to be Gotham 4r, 14px? Need to change my spec, they're just way too small. Please update for all components that use it?

image

**Comment by joshlory** [comment]
Are the images scaled proportionally?  It looks like the Announcement image is squashed horizontally and the Course images are stretched horizontally.

**Pull request merged**